### PR TITLE
0.18.0 Changelog

### PR DESCRIPTION
**Pull request by tatlat**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
 {{% /alert %}}
 
 [Unreleased]
+### Added
+- etcd encryption for CloudStack and Vsphere: [#6557](https://github.com/aws/eks-anywhere/pull/6557)
+- [KINDless upgrades](https://github.com/aws/eks-anywhere/blob/main/designs/kindless-upgrades.md)
```
Comment (vivek-koppuru) [inline docs/content/en/docs/whatsnew/changelog.md:21]: @panktishah26 Do we have an issue for this instead to link?

File: docs/content/en/docs/whatsnew/changelog.md [hunk 2]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
+- Support for modular Kubernetes version upgrades for bare metal: [#6735](https://github.com/aws/eks-anywhere/pull/6735)
+  - OSImageURL added to Tinkerbell Machine Config
+- Bare metal [out-of-band](https://github.com/aws/eks-anywhere/blob/main/designs/bare-metal-out-of-band-webhook.md) webhook
+- Support for Kubernetes 1.28
+- Support for air gapped image building: [#6457](https://github.com/aws/eks-anywhere/issues/6457)
+
+
+### Changed
+- etcd reconciliation separated from kcp: [#6496](https://github.com/aws/eks-anywhere/pull/6496)
+- Upgraded cilium to `1.12.15`
```
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:32]:
```suggestion
- Upgraded Cilium to `1.12.15`
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 3]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
+- Generate TinkerbellTemplateConfig command: [#3588](https://github.com/aws/eks-anywhere/issues/3588)
+- Support for modular Kubernetes version upgrades for bare metal: [#6735](https://github.com/aws/eks-anywhere/pull/6735)
+  - OSImageURL added to Tinkerbell Machine Config
+- Bare metal [out-of-band](https://github.com/aws/eks-anywhere/blob/main/designs/bare-metal-out-of-band-webhook.md) webhook
+- Support for Kubernetes 1.28
+- Support for air gapped image building: [#6457](https://github.com/aws/eks-anywhere/issues/6457)
+
+
+### Changed
+- etcd reconciliation separated from kcp: [#6496](https://github.com/aws/eks-anywhere/pull/6496)
```
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:31]:
```suggestion
- Etcd reconciliation separated from KubernetesControlPlane: [#6496](https://github.com/aws/eks-anywhere/pull/6496)
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 4]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
+  - `v1-27-eks-12` to [`v1-27-eks-13`](https://distro.eks.amazonaws.com/releases/1-27/13/)
+  - [`v1-28-eks-26`](https://distro.eks.amazonaws.com/releases/1-28/6/)
+- Cluster API Provider Cloudstack: `v0.4.9-rc6` to `v0.4.9-rc7`
+- Cluster API Provider AWS Snow: `v0.1.26` to `v0.1.27`
+
+### Removed
+- Support for Kubernetes 1.23
+
+### Fixed
+- Fail on eksctl anywhere upgrade cluster plan -f: [#6716](https://github.com/aws/eks-anywhere/issues/6716)
```
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:46]:
```suggestion
- Fail on `eksctl anywhere upgrade cluster plan -f`: [#6716](https://github.com/aws/eks-anywhere/issues/6716)
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 5]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
 {{% /alert %}}
 
 [Unreleased]
+### Added
+- etcd encryption for CloudStack and Vsphere: [#6557](https://github.com/aws/eks-anywhere/pull/6557)
```
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:20]:
```suggestion
- Etcd encryption for CloudStack and vSphere: [#6557](https://github.com/aws/eks-anywhere/pull/6557)
```

File: docs/content/en/docs/whatsnew/changelog.md (hunk 1 above)
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:21]:
```suggestion
- [KinD-less upgrades](https://github.com/aws/eks-anywhere/blob/main/designs/kindless-upgrades.md)
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 6]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
+- [KINDless upgrades](https://github.com/aws/eks-anywhere/blob/main/designs/kindless-upgrades.md)
+- Generate TinkerbellTemplateConfig command: [#3588](https://github.com/aws/eks-anywhere/issues/3588)
+- Support for modular Kubernetes version upgrades for bare metal: [#6735](https://github.com/aws/eks-anywhere/pull/6735)
```
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:23]:
```suggestion
- Support for modular Kubernetes version upgrades with bare metal: [#6735](https://github.com/aws/eks-anywhere/pull/6735)
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 7]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
+### Changed
+- etcd reconciliation separated from kcp: [#6496](https://github.com/aws/eks-anywhere/pull/6496)
+- Upgraded cilium to `1.12.15`
+- Upgraded EKS-D:
+  - `v1-24-eks-26` to [`v1-24-eks-27`](https://distro.eks.amazonaws.com/releases/1-24/27/)
+  - `v1-25-eks-22` to [`v1-25-eks-23`](https://distro.eks.amazonaws.com/releases/1-25/23/)
+  - `v1-26-eks-18` to [`v1-26-eks-19`](https://distro.eks.amazonaws.com/releases/1-26/19/)
+  - `v1-27-eks-12` to [`v1-27-eks-13`](https://distro.eks.amazonaws.com/releases/1-27/13/)
+  - [`v1-28-eks-26`](https://distro.eks.amazonaws.com/releases/1-28/6/)
+- Cluster API Provider Cloudstack: `v0.4.9-rc6` to `v0.4.9-rc7`
```
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:39]:
```suggestion
- Cluster API Provider CloudStack: `v0.4.9-rc6` to `v0.4.9-rc7`
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 8]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
+- Cluster API Provider AWS Snow: `v0.1.26` to `v0.1.27`
+
+### Removed
+- Support for Kubernetes 1.23
```
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:43]:
```suggestion
- Support for Kubernetes v1.23
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 9]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
 
 [Unreleased]
+### Added
+- etcd encryption for CloudStack and Vsphere: [#6557](https://github.com/aws/eks-anywhere/pull/6557)
+- [KINDless upgrades](https://github.com/aws/eks-anywhere/blob/main/designs/kindless-upgrades.md)
+- Generate TinkerbellTemplateConfig command: [#3588](https://github.com/aws/eks-anywhere/issues/3588)
+- Support for modular Kubernetes version upgrades for bare metal: [#6735](https://github.com/aws/eks-anywhere/pull/6735)
+  - OSImageURL added to Tinkerbell Machine Config
+- Bare metal [out-of-band](https://github.com/aws/eks-anywhere/blob/main/designs/bare-metal-out-of-band-webhook.md) webhook
+- Support for Kubernetes 1.28
```
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:26]:
```suggestion
- Support for Kubernetes v1.28
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 10]
```diff
@@ -16,6 +16,50 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
 {{% /alert %}}
 
 [Unreleased]
+### Added
+- etcd encryption for CloudStack and Vsphere: [#6557](https://github.com/aws/eks-anywhere/pull/6557)
+- [KINDless upgrades](https://github.com/aws/eks-anywhere/blob/main/designs/kindless-upgrades.md)
+- Generate TinkerbellTemplateConfig command: [#3588](https://github.com/aws/eks-anywhere/issues/3588)
+- Support for modular Kubernetes version upgrades for bare metal: [#6735](https://github.com/aws/eks-anywhere/pull/6735)
+  - OSImageURL added to Tinkerbell Machine Config
+- Bare metal [out-of-band](https://github.com/aws/eks-anywhere/blob/main/designs/bare-metal-out-of-band-webhook.md) webhook
```
Comment (vivek-koppuru) [inline docs/content/en/docs/whatsnew/changelog.md:25]: @jacobweinstock same with this, do we have an issue?

**Review by chrisdoherty4 — APPROVED** [review]

File: docs/content/en/docs/whatsnew/changelog.md [hunk 11]
```diff
@@ -15,7 +15,49 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
+### Added
+- Etcd encryption for CloudStack and vSphere: [#6557](https://github.com/aws/eks-anywhere/pull/6557)
+- KinD-less upgrades: [#6622](https://github.com/aws/eks-anywhere/pull/6622)
+- Generate TinkerbellTemplateConfig command: [#3588](https://github.com/aws/eks-anywhere/issues/3588)
+- Support for modular Kubernetes version upgrades with bare metal: [#6735](https://github.com/aws/eks-anywhere/pull/6735)
+  - OSImageURL added to Tinkerbell Machine Config
+- Bare metal out-of-band webhook: [#5738](https://github.com/aws/eks-anywhere/issues/5738)
+- Support for Kubernetes v1.28
+- Support for air gapped image building: [#6457](https://github.com/aws/eks-anywhere/issues/6457)
+
```
Comment (abhinavmpandey08) [inline docs/content/en/docs/whatsnew/changelog.md:38]: - Support for RHEL 8 and RHEL 9 for Nutanix provider: [#6822](https://github.com/aws/eks-anywhere/pull/6822)

**Review by abhinavmpandey08 — APPROVED** [review]
/lgtm
/woof

**Pull request merged**